src/components/CivicProfileForms/BasicInfo.jsx

```diff
@@ -1,8 +1,181 @@
 // React Imports
-import React from 'react';
-// MUI Imports
-import Typography from '@mui/material/Typography';
+import React, { useState, useEffect } from 'react';
+// Material UI Imports
+import Button from '@mui/material/Button';
+import CheckIcon from '@mui/icons-material/Check';
+import ClearIcon from '@mui/icons-material/Clear';
+import FormControl from '@mui/material/FormControl';
+import FormHelperText from '@mui/material/FormHelperText';
+import Grid from '@mui/material/Grid';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import TextField from '@mui/material/TextField';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import dayjs from 'dayjs';
+// Hook Imports
+import { useCivicProfile } from '@hooks';
+// Component Imports
+import { FormSection } from '../Form';
 
-const BasicInfo = () => <Typography>Basic Info</Typography>;
+/**
+ * BasicInfo Component - A form to fill out basic user info
+ *
+ * @memberof CivicProfileForms
+ * @name BasicInfo
+ * @returns {React.JSX.Element} The BasicInfo Component
+ */
+const BasicInfo = () => {
+  const { data, add, isSuccess, storedDataset, refetch } = useCivicProfile();
+  const [formData, setFormData] = useState({
+    firstName: '',
+    lastName: '',
+    dateOfBirth: null,
+    gender: ''
```

Comment:
Not too big of an issue since the feature appears to be working as intended, however, MUI is complaining about the input fields if it's not using one of the values from the option elements.

<img width="800" alt="Screenshot 2024-04-23 at 7 45 27 PM" src="https://github.com/codeforpdx/PASS/assets/14917816/0916692a-292c-40e1-a06e-e9ac00264449">
<img width="800" alt="Screenshot 2024-04-23 at 7 45 41 PM" src="https://github.com/codeforpdx/PASS/assets/14917816/5c47c871-5db9-46dc-ac8c-b9a802de3a79">

Think we could simply set the defaultValue, state, and clear value for gender as `9` to work around the problem. So Line 36, Line 75, and Line 133.